HacktoberfestProject.Web/Controllers/HomeController.cs

```diff
@@ -33,7 +34,7 @@ public async Task<IActionResult> Index()
             User user = null;
             if (_contextAccessor.HttpContext.User.Identity.IsAuthenticated)
             {
-                var username = _contextAccessor.HttpContext.User.Claims.FirstOrDefault(c => c.Type == GitHubUsernameClaimType).Value;
+                var username = _contextAccessor.HttpContext.User.Claims.FirstOrDefault(c => c.Type == GitHubUsernameClaimType)?.Value;

```

Comment:
What do you do if `username` is null?

Reply:
That is another question I have, what would we like an exception here/error page?

Reply:
see below